### PR TITLE
Feature/issue 6 submission function

### LINKED ISSUE #6: Implement submission function
Details to follow

### PR DESCRIPTION
**Pull request by Chen2x**
Created a submission function for making post requests to the testbed api

### CODE REVIEW

File: ga4gh/testbed/submit/submit_to_api.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import requests
+
+
+class Submit():
```
Comment (jb-adams) [inline ga4gh/testbed/submit/submit_to_api.py]: could be more descriptive, maybe `ReportSubmitter`?

File: ga4gh/testbed/submit/submit_to_api.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+    def __init__(self, url, series_id, series_token, report) :
+        self.url = url
+        self.series_id = series_id
+        self.series_token = series_token
+        self.report = report
+
+    def submit_report(self):
```
Comment (jb-adams) [inline ga4gh/testbed/submit/submit_to_api.py]: please add docstring for the method

**Review by jb-adams — CHANGES_REQUESTED** [review]
Looks good so far. I have the following suggestions:
* To make this even simpler, you could remove the `__init__` constructor and make `submit_report` a static method that accepts a url, series id, series token, and report
* make `url` a keyword argument, with a default value of the test endpoint, e.g.
```
def submit_report(..., url="http://localhost:4500/reports")
```

There's also no test code for this module. When you add your unit tests, assume that there is a running testbed api at `http://localhost:4500`, and that it has the two test report series we discussed

**Review by jb-adams — CHANGES_REQUESTED** [review]
In the last review, I mentioned:

> There's also no test code for this module. When you add your unit tests, assume that there is a running testbed api at http://localhost:4500, and that it has the two test report series we discussed

I'm still not seeing any test code, which means there's no way to know if the function works correctly or breaks in the future. Can you please add tests?

**Comment by Chen2x** [comment]
Whoops I missed that part. Adding the tests in today.

File: ga4gh/testbed/submit/report_submitter.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        
+        Required arguments:
+            series_id - A series ID is needed by server to group the report
+            series_token - A token is needed to verify authenticity
+            report - GA4GH report in JSON format
+            url - URL of the testbed server
+        '''
+        header = {"GA4GH-TestbedReportSeriesId": series_id, "GA4GH-TestbedReportSeriesToken": series_token}
+        submit_request = requests.post(url, headers=header ,json=report)
+        return submit_request.status_code
```
Comment (jb-adams) [inline ga4gh/testbed/submit/report_submitter.py]: Returning the status code from this method is insufficient, especially if there's been an error we want to know why (i.e. the message). If the submission has been successful, then we want the `id` of the new report that was created on the server. So,

- Always return status code
- Return error message if submission wasn't successful, `None` if submission was successful
- Return new report id if submission was successful, `None` if submission wasn't successful

Should be pretty easy to do in a Python dictionary

**Review by jb-adams — CHANGES_REQUESTED** [review]
in addition to the inline comments, the failing tests should also be addressed

**Review by jb-adams — APPROVED** [review]

**Pull request merged**